src/storage/client/StorageClient.inl

```diff
@@ -158,6 +158,49 @@ folly::SemiFuture<StorageRpcResponse<Response>> StorageClient::collectResponse(
     return context->promise.getSemiFuture();
 }
 
+
+template<class Request, class RemoteFunc, class Response>
+folly::Future<StatusOr<Response>> StorageClient::getResponse(
+        folly::EventBase* evb,
+        std::pair<HostAddr, Request> request,
+        RemoteFunc remoteFunc) {
+    folly::Promise<StatusOr<Response>> pro;
+    auto f = pro.getFuture();
+    if (evb == nullptr) {
+        DCHECK(!!ioThreadPool_);
+        evb = ioThreadPool_->getEventBase();
+    }
+    auto host = request.first;
+    LOG(INFO) << "Send request to storage " << host;
+    auto client = clientsMan_->client(host, evb);
+    auto spaceId = request.second.get_space_id();
+    remoteFunc(client.get(), std::move(request.second))
```

Comment:
remoteFunc will be running in evb,   otherwise it maybe crashed.  
Please refer to MetaClient getResponse.

Reply:
That's good :)